<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDialog</class>
 <widget class="QDialog" name="SelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QGroupBox" name="select">
       <property name="title">
        <string>Select</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QCheckBox" name="sameSubtype">
          <property name="text">
           <string>Same subtype:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLabel" name="type">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="sameStaff">
          <property name="text">
           <string>Same staff</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="sameVoice">
          <property name="text">
           <string>Same voice</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="sameDuration">
          <property name="text">
           <string>Same duration</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLabel" name="subtype">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Element type:</string>
          </property>
          <property name="buddy">
           <cstring>type</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="sameBeat">
          <property name="text">
           <string>Same beat</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="sameMeasure">
          <property name="text">
           <string>Same measure</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="sameSystem">
          <property name="text">
           <string>Same system</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="inSelection">
          <property name="text">
           <string>In selection</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="action">
       <property name="title">
        <string>Action</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" rowspan="2">
         <widget class="QRadioButton" name="replace">
          <property name="text">
           <string>Replace selection</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="add">
          <property name="text">
           <string>Add to selection</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="fromSelection">
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Search in selection</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="subtract">
          <property name="text">
           <string>Subtract from selection</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>type</tabstop>
  <tabstop>sameSubtype</tabstop>
  <tabstop>subtype</tabstop>
  <tabstop>sameStaff</tabstop>
  <tabstop>sameVoice</tabstop>
  <tabstop>inSelection</tabstop>
  <tabstop>sameDuration</tabstop>
  <tabstop>sameSystem</tabstop>
  <tabstop>sameBeat</tabstop>
  <tabstop>sameMeasure</tabstop>
  <tabstop>replace</tabstop>
  <tabstop>add</tabstop>
  <tabstop>subtract</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
